src/rules/html/warn-on-remote-script.js

```diff
@@ -0,0 +1,25 @@
+import { VALIDATION_WARNING } from 'const';
+import { isStrictRelativeUrl } from 'schema/formats';
+import * as messages from 'messages';
+
+
+export function warnOnRemoteScript($, filename) {
+  return new Promise((resolve) => {
+    var linterMessages = [];
+
+    $('script').each((i, element) => {
+      const src = $(element).attr('src');
+
+      if (src !== undefined && !isStrictRelativeUrl(src)) {
+        linterMessages.push(
+          Object.assign({}, messages.REMOTE_SCRIPT, {
+            type: VALIDATION_WARNING,
```

Comment:
We haven't made any exceptions in far more than a year, and I'd like to keep it that way, as I don't think we should make any exceptions at all. It nullifies the purpose of signing. I believe @kewisch agrees.